src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -78,6 +78,9 @@ internal interface ISyntaxFactsService : ILanguageService
 
         bool IsBinaryExpression(SyntaxNode node);
         void GetPartsOfBinaryExpression(SyntaxNode node, out SyntaxNode left, out SyntaxNode right);
```

Comment:
this can be moved to be in ISyntaxFactsExtensions now.

Reply:
How do we determine when to move a method to extensions?